### PR TITLE
Add Drupal at your Fingertips book to PHP/Drupal section

### PR DESCRIPTION
**Pull request by selwynpolit**
## What does this PR do?
Add Drupal book

### How do we know it's really free?
I wrote it.  There are no limitations to accessing it

### For book lists, is it a book? For course lists, is it a course? etc.
It is a book

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -1830,6 +1830,7 @@ Books on general-purpose programming that don't focus on a specific language are
 
 #### Drupal
 
+* [Drupal at your Fingertips](https://selwynpolit.github.io/d9book/) - Selwyn Polit and Drupal Community Contributors
```
Comment (eshellman) [inline books/free-programming-books-langs.md:1833]:
```suggestion
* [Drupal at your Fingertips](https://selwynpolit.github.io/d9book/) - Selwyn Polit, Drupal Community Contributors
```

Great addition!

**Comment by oguzhan18** [comment]
This is an automated response to help you with this issue. Please let us know if you need further assistance.

File: books/free-programming-books-langs.md (hunk 1 above)
Comment (selwynpolit) [reply books/free-programming-books-langs.md:1833]: @eshellman Any idea how to get this PR merged?

Comment (eshellman) [reply books/free-programming-books-langs.md:1833]: commit the suggestion!

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**